src/super_gradients/training/models/predictions.py

```diff
@@ -0,0 +1,75 @@
+from typing import Tuple
+from abc import ABC
+from dataclasses import dataclass
+
+import numpy as np
+
+from super_gradients.common.factories.bbox_format_factory import BBoxFormatFactory
+from super_gradients.training.datasets.data_formats.bbox_formats import convert_bboxes
+
+
+@dataclass
+class Prediction(ABC):
+    pass
+
+
+@dataclass
+class DetectionPrediction(Prediction):
+
+    _bboxes: np.ndarray
+    _bbox_format: str
+
+    confidence: np.ndarray
+    labels: np.ndarray
+    image_shape: Tuple[int, int]
+
+    def __init__(self, bboxes: np.ndarray, bbox_format: str, confidence: np.ndarray, labels: np.ndarray, image_shape: Tuple[int, int]):
+        """
+        :param bboxes:      BBoxes in the format specified by bbox_format
+        :param bbox_format: BBoxes format that can be a string ("xyxy", "cxywh", ...)
+        :param confidence:  Confidence scores for each bounding box
+        :param labels:      Labels for each bounding box
+        :param image_shape: Shape of the image the prediction is made on
+        """
+        self._bboxes = bboxes
+        self._bbox_format = bbox_format
```

Comment:
Why don't we directly convert and store boxes in XYXY format?
For me it sounds perfectly fine to drop the original predictions format from the model and narrow the scope only to XYXY.

Reply:
Sounds reasonable

Reply:
35717803ed9636d6a04d886010f078f532783c1b
I am not sure about bboxes naming.
I tend to prefer to be explicit about format (`bboxes_xyxy`) because I see people missing that we convert the bbox to xyxy inside the init.

Reply:
👍 for `bboxes_xyxy` name